File: Portfolio/portfolio/src/app/entidades/persona.ts
export class Persona{

 private _id: number;
 private _fullName: string;
 private _position: string;
 private _ubication: string;
 private _url: string;
 private _background: string;

//getters and setters
public get id(): number {
  return this._id;
}
public set id(value: number) {
  this._id = value;
}
public get fullName(): string {
  return this._fullName;
}
public set fullName(value: string) {
  this._fullName = value;
}
public get position(): string {
  return this._position;
}
public set position(value: string) {
  this._position = value;
}
public get ubication(): string {
  return this._ubication;
}
public set ubication(value: string) {
  this._ubication = value;
}
public get url(): string {
  return this._url;
}
public set url(value: string) {
  this._url = value;
}
public get background(): string {
  return this._background;
}
public set background(value: string) {
  this._background = value;
}


 constructor( id:number , fullName:string,   position:string, ubication:string, url:string, background: string)
 {
     this._id=id;
     this._fullName=fullName;
     this._position=position;
     this._ubication=ubication;
     this._url=url;
     this._background=background;
 }

}

File: Java_sprintboot/portfolio/src/main/java/com/portfolioWeb/portfolio/Model/Persona.java

package com.portfolioWeb.portfolio.Model;
import javax.persistence.Basic;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import lombok.Getter;
import lombok.Setter;

@Getter @Setter
@Entity
public class Persona {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
 private Long id;
    @Basic
 private String fullName ;
 private String position ;
 private String ubication;
 private String url ;
 private String background;

    public Persona(Long id, String fullName, String position, String ubication, String url, String background) {
        this.id = id;
        this.fullName = fullName;
        this.position = position;
        this.ubication = ubication;
        this.url = url;
        this.background = background;
    }



    public Persona() {
    }

   
 
    
}
